examples/ring-spec-swagger/test/example/server_test.clj

```diff
@@ -0,0 +1,31 @@
+(ns example.server-test
+  (:require [clojure.test :refer :all]
+            [example.server :refer [app]]))
+
+(deftest example-server
+
+  (testing "GET"
+    (is (= (-> {:request-method :get :uri "/math/plus" :query-params {:x 20 :y 3}}
```

Comment:
`:query-params` are received as string keys and string values from parameters mw. e.g. `{"x" "20", "y" "3"}`. To test that mw too, there could just be `:query-string` of `"x=30&y=3"` (as received from the adapter)